docker-compose-dev.yml

```diff
@@ -18,3 +18,11 @@ services:
     image: redis:3.0
     ports:
       - "6379:6379"
+  inkpy:
+    image: ar4s/inkpy_jinja:latest
```

Comment:
use allegro's image: https://hub.docker.com/r/allegro/inkpy

Reply:
just did it.